pkgs/applications/graphics/inkscape/default.nix

```diff
@@ -184,12 +169,16 @@ stdenv.mkDerivation rec {
 
   passthru.tests.ps2pdf-plugin = callPackage ./test-ps2pdf-plugin.nix { };
```

Comment:
Might be useful to add `python3.pkgs.inkex` to `passthru.tests` as well.

Reply:
Good idea. Is `python3Packages.inkex` fine too? `python3Packages` seems to be preferred over `python3.pkgs`.